Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.4
```

Comment:
I recommend testing this change on Mapbox Directions directly to make sure there are no regressions.

Reply:
I tried MapboxDirections with this dependency and it resolved and compiled correctly.